WIP Implemented START STOP qualifiers support for Splunk SPL query

bhaveshkr opened the pull request.

bhaveshkr committed:
Implemented START STOP qualifiers support for Splunk SPL query

samcornwell commented on stix2patterns_translator/search_platforms/splunk_search.py:
```diff
@@ -32,24 +34,58 @@ def __init__(self, pattern:Pattern, data_model_mapper, object_scoper = object_sc
+                latest_obj = datetime.strptime(latest, stix_date_format)
+                latest_dt = latest_obj.strftime(splunk_date_format)[:-3]+'Z'
+                # prepare splunk SPL query
+                return "{query_string} |earliest={earliest} |latest={latest}".format(query_string=translated_query_str, 
```
Is it correct to have the pipes between `earliest` and `latest`?

bhaveshkr committed:
refactored qualifier code for Splunk, added field object in x509-certificate CIM mapping and binary_ref fields in process mapping.

bhaveshkr commented:
I have refactored splunk qualifier code. I have added condition to check qualifier if it has just START or STOP  single patameter but at the moment qualifier object expects both values to be passed. Also, I wrapped existing fields of x509-certificate to field object because it was breaking.
